Makefile

```diff
@@ -483,6 +484,26 @@ test-mssql-migration: migrations.mssql.test migrations.individual.mssql.test gen
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.mssql.test -test.failfast
 	GITEA_ROOT="$(CURDIR)" GITEA_CONF=integrations/mssql.ini ./migrations.individual.mssql.test -test.failfast
 
+.PHONY: test-e2e
+test-e2e: test-e2e-pgsql
+
+.PHONY: test-e2e\#%
+test-e2e\#%: test-e2e-pgsql\#%
+	echo "Why do I need this?"
```

Comment:
These are just because make does not support passing arguments to targets, but I think we should just document the appropriate `npx jest <something>` command to run single tests. I also wonder if we could just launch `jest` to run the actual tests, so we never need to `npx playwright`.

Reply:
Ah, it's because make... I wanted to keep it similar to the integration tests, but I can get rid of this if you really want.